config/jobs/kubernetes-csi/external-provisioner/external-provisioner-manual-job-config.yaml

```diff
@@ -0,0 +1,56 @@
+# Derived from external-provisioner-config.yaml, modified and maintained manually.
+
+presubmits:
+  kubernetes-csi/external-provisioner:
+  - name: pull-kubernetes-csi-external-provisioner-distributed-on-kubernetes-master
+    # Explicitly needs to be started with /test.
+    # This cannot be enabled by default because there's always the risk
+    # that something changes in master which breaks the pre-merge check.
+    always_run: false
+    optional: true
+    decorate: true
+    skip_report: false
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-bazel-remote-cache-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    annotations:
+      testgrid-dashboards: sig-storage-csi-external-provisioner
+      testgrid-tab-name: distributed-on-kubernetes-master
+      description: Kubernetes-CSI pull job in repo external-provisioner for distributed provisioning, with CSIStorageCapacity
+    spec:
+      containers:
+      # We need this image because it has Docker in Docker and go.
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210226-c001921-master
+        command:
+        - runner.sh
+        args:
+        - ./.prow.sh
+        env:
+        - name: CSI_PROW_KUBERNETES_VERSION
+          value: "latest"
+        - name: CSI_PROW_USE_BAZEL
+          value: "false"
+        - name: CSI_PROW_DRIVER_VERSION
+          value: "v1.5.0"
+        - name: CSI_SNAPSHOTTER_VERSION
+          value: "master"
+        - name: CSI_PROW_TESTS
+          value: "sanity serial parallel serial-alpha parallel-alpha"
+        - name: CSI_PROW_SANITY_POD
+          value: csi-hostpath-socat-0
+        - name: CSI_PROW_SANITY_CONTAINER
+          value: socat
+        - name: CSI_PROW_DEPLOYMENT
+          value: "kubernetes-distributed"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+      resources:
+        requests:
+          # these are both a bit below peak usage during build
+          # this is mostly for building kubernetes
+          memory: "9000Mi"
```

Comment:
I see this memory setting is removed from other places, but kept here.  Can you explain why?

Reply:
The gen script for the other jobs uses a heuristic to determine whether Kubernetes must be built from source: the unstable Kubernetes is assumed to be built from source, everything else is assumed to be supported by KinD images. That heuristic is currently wrong, 1.20 still gets built from source. I'm changing that in https://github.com/kubernetes-csi/csi-release-tools/pull/138 so it didn't seem worthwhile to override the heuristic.

Here I kept the higher memory requirement because these jobs will be updated soonish anyway - I want to move them into the automatically generated jobs.